Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -257,7 +257,8 @@ def load_communities():
 
         self.initComplete = True
 
-    def add(self, tdef, dscfg, pstate=None, initialdlstatus=None, setupDelay=0, hidden=False):
+    def add(self, tdef, dscfg, pstate=None, initialdlstatus=None, setupDelay=0, hidden=False,
```

Comment:
the setupDelay (and other arguments) also already like that, even in the devel branch https://github.com/Tribler/tribler/blob/devel/Tribler/Core/APIImplementation/LaunchManyCore.py#L266.